Classes/Controller/SuggestController.php

```diff
@@ -0,0 +1,197 @@
+<?php
+declare(strict_types=1);
+
+namespace PunktDe\Neos\AdvancedSearch\Controller;
+
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Eel\ElasticSearchQueryBuilder;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\ElasticSearchClient;
+use Flowpack\ElasticSearch\ContentRepositoryAdaptor\Exception\QueryBuildingException;
+use Flowpack\SearchPlugin\Suggestion\SuggestionContextInterface;
+use Flowpack\SearchPlugin\Utility\SearchTerm;
+use Neos\Cache\Frontend\VariableFrontend;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Mvc\Controller\ActionController;
+use Neos\Flow\Mvc\View\JsonView;
+use Neos\Flow\Persistence\Exception\IllegalObjectTypeException;
+use Neos\Neos\Controller\CreateContentContextTrait;
+
+class SuggestController extends ActionController
+{
+    use CreateContentContextTrait;
+
+    protected ElasticSearchClient $elasticSearchClient;
+
+    /**
+     * @var ElasticSearchQueryBuilder
+     */
+    protected ElasticSearchQueryBuilder $elasticSearchQueryBuilder;
```

Comment:
```suggestion
    protected ?ElasticSearchQueryBuilder $elasticSearchQueryBuilder;
```

Ist this working? The variable is not assigned in the constructor.